test/sharness/t0040-add-and-cat.sh

```diff
@@ -379,4 +379,11 @@ test_expect_success "ipfs cat file fails" '
 
 test_add_named_pipe ""
 
+# Test daemon in offline mode
+test_launch_ipfs_daemon --offline
```

Comment:
great. i think we want to test `--offline` with almost every other test too. We don't want nasty surprises.